.changeset/spicy-buckets-own.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": minor
+---
+
+Added dependency on `jotai`.
```

Comment:
Yes. The consumer (application) needs to be aware of which dependencies we require, because they are the ones managing all deps in the end. One example of an effect: If they already use `jotai` in their application, they will likely want to make sure that only one copy of `jotai` is included in the final client bundle.